cmd/kyma/alpha/create/module/module.go

```diff
@@ -61,6 +61,11 @@ func (c *command) Run(args []string) error {
 		cli.AlphaWarn()
 	}
 
+	_, err := module.ValidateModuleName(args[0])
```

Comment:
This kind of stutters, I would suggest calling it `module.ValidateName()`. Package name always gives context and has to be taken into consideration when naming types and functions.

Reply:
👍